modules/administration/pages/image-management.adoc

```diff
@@ -403,24 +404,39 @@ Disk space depends on the actual size of the image.
 For more information, see the documentation of the underlying system.
 ====
 
+[WARNING]
+====
+The build host must be a Salt client.
+Do not install the build host as a traditionally managed client.
```

Comment:
```suggestion
Do not install the build host as a traditional client.
```
Now, we call these beasts just "traditional client". Or is the longer version also fine, @Loquacity ?

Reply:
I prefer shorter.